File: program/program/convert-hours-to-seconds/ConvertHoursToSeconds.cs
using System;

class MainClass {
    public static void Main (string[] args) {
    Console.WriteLine ("Please enter the hours : ");
    double hours = Convert.ToDouble(Console.ReadLine());
    double seconds = hours * 3600;
    Console.WriteLine(hours+" hours is "+seconds+" seconds");
  }
}

File: program/program/convert-hours-to-seconds/ConvertHoursToSeconds.kt
fun main() {
    print("Enter weeks : ")
    println()
    var input = readLine()

    try {
        println("$input hours = " + ConvertHoursToSeconds(input!!.toInt()) + " seconds")
    } catch (exception: Exception) {
        println("Input string was not in a correct format")
    }

}

private fun ConvertHoursToSeconds(hr: Int): Int {
    val hoursToSeconds = hr * 3600

    return  hoursToSeconds
}